swo_aws_extension/flows/jobs/billing_journal/billing_journal_generator.py

```diff
@@ -275,14 +287,52 @@ def _generate_agreement_journal_lines(self, agreement):
             mpa_account,
             first_active_subscription,
             aws_client,
-            organization_reports,
-            organization_invoices,
             journal_details,
         )
         self._log(
             "info",
             f"Generated journal lines for organization account: {mpa_account}",
         )
+        self._add_attachments(agreement.get("id"), journal_id, mpa_account)
+
+    def _add_attachments(self, agreement_id, journal_id, mpa_account):
+        """
+        Adds attachments to the journal
+        Args:
+            agreement_id (str): Agreement ID used to name the attachment.
+            journal_id (str): Journal ID to which the attachments will be added.
+            mpa_account (str): AWS organization account ID.
+        Returns:
+            None
+        """
```

Comment:
Please, check the expected style for docstrings.  We are missing some blank lines and puntuaction. It's better to add properly now, otherwise we will deal with it when the new rules are applied. ☺️

Reply:
Or Pasha is going to fix it :-D

Reply:
Let's do it in next PR :-)